core/modules/widgets/action-listops.js

```diff
@@ -3,87 +3,89 @@ title: $:/core/modules/widgets/action-listops.js
 type: application/javascript
 module-type: widget
 
-Action widget to apply list operations to any tiddler field (defaults to the 'tags' field of the current tiddler)
+Action widget to apply list operations to any tiddler field (defaults to the 'list' field of the current tiddler)
 
 \*/
 (function () {
 
-  /*jslint node: true, browser: true */
-  /*global $tw: false */
-  "use strict";
+    /*jslint node: true, browser: true */
+    /*global $tw: false */
+    "use strict";
 
-  var Widget = require("$:/core/modules/widgets/widget.js").widget;
+    var Widget = require("$:/core/modules/widgets/widget.js")
+        .widget;
 
-  var ActionListopsWidget = function (parseTreeNode, options) {
-    this.initialise(parseTreeNode, options);
-  };
+    var ActionListopsWidget = function (parseTreeNode, options) {
+        this.initialise(parseTreeNode, options);
```

Comment:
My editor (Sublime Text) is set to use spaces as jslint throws up multiple errors if I do not do so, and I like the genuine errors to be easily identifiable. There will be no problem changing the spaces to tabs -- my problem is working with Github. I suppose I can write over the entire file with the corrected indents.